packages/slice-machine/src/tracker.ts

```diff
@@ -219,17 +219,20 @@ export class SMTracker {
 
   async trackCustomTypeFieldAdded({
     fieldId,
+    fieldLabel,
     customTypeId,
     zone,
     type,
   }: {
     fieldId: string;
+    fieldLabel: string;
     customTypeId: string;
     zone: "static" | "repeatable";
     type: string;
   }): Promise<void> {
     const data = {
       id: fieldId,
+      label: fieldLabel,
```

Comment:
not sure how the tracking event works, is it fine to add an extra parameter to the data?

Reply:
yes and no, yes it's fine but we'll need to and an extra parameter to the validation in segment :)

Reply:
anything I need to add?